natgeo/mortar
# [SYTO-1075] Gives the CTA icon some right padding at the mobile breakpoint

https://jira.natgeo.com/browse/SYTO-1075

Small tweak for https://jira.natgeo.com/browse/SYTO-1075 to give CTA container padding so the icon doesn't bump against the viewport in mobile

To test:

- Pull SYTO-1075-3
- gulp serve 
- head over to http://localhost:3000/components/podpromo/
- Hit mobile breakpoint
- Make the CTA text really long
- Verify there's 30px padding between + icon and right edge of image
